tests/lib/util/npm-util.js

```diff
@@ -66,6 +67,27 @@ describe("npmUtil", function() {
             fs.existsSync.restore();
             fs.readFileSync.restore();
         });
+
+        it("should throw with message when parsing invalid package.json", function() {
+            var logInfo = sinon.stub(log, "info");
```

Comment:
If we use `sinon.sandbox.create()` in test setup and `sandbox.restore()` in test teardown, then we don't need to worry about restoring at the end of a test (and as a bonus it is exception safe as well). Just a suggestion :smile: